rhobs/alerting/data_plane/prometheus.serviceprovider_alerts.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-spi-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: serviceprovider_alerts
+      interval: 1m
+      rules:
+        - alert: ServiceProvider5xxErrorsRate
+          expr: |
+            (
+              sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{status=~"5.."}[1h]))
+              or sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count[1h])) * 0
+            )
+            /
+            sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count[1h])) > 0.10
+          for: 5m
+          labels:
+            severity: warning
+            slo: true
+          annotations:
+            summary: >-
+              SPI is experiencing 5xx errors from service provider {{ $labels.sp }} of >10%
+            description: >-
+              Application controller in Pod {{ $labels.pod }} for namespace
+              {{ $labels.namespace }} on instance {{ $labels.source_cluster }} having a
+              {{  printf "%.2f" $value }} ratio of 5xx errors from service provider {{ $labels.sp }} for latest 60 minutes
```

Can use this (learned it yesterday :) ):
```suggestion
              {{ $value | humanizePercentage }}% of 5xx errors from service provider {{ $labels.sp }} for latest 60 minutes
```